test/tests.py

```diff
@@ -188,6 +188,11 @@ def assert_metrics(self, expected_metrics: Dict[str, List[str]], lookback_minute
         logger.debug(f"Waiting {wait_before_check_seconds} before checking for metrics.")
         time.sleep(wait_before_check_seconds)
         for component, expected_comp_metrics in expected_metrics.items():
+            if component == "captureProxy" and self.deployment_type == "cloud":
+                # We currently do not emit captureProxy metrics from a non-standalone proxy, which is the scenario
```

Comment:
Thanks for comment here, a log statement that metric assertions are being skipped could also bring clarity to say users looking at this test through Jenkins logs

Reply:
I like that, also a good reminder to change this in the future.